doc/ProfilesJSONDocumentation.md

```diff
@@ -0,0 +1,57 @@
+# Profiles.json Documentation
+
+| Property | Necessity | Type | Description |
+| -------- | ---- | ----------- | ----------- |
+| **Global** |  |  | **Properties listed below affect the entire window, regardless of the property settings.** |
+| `alwaysShowTabs` | _Required_ | Boolean | When set to `true`, tabs are always displayed. When set to `false` and `experimental_showTabsInTitlebar` is set to `false`, tabs only appear after typing <kbd>Ctrl</kbd> + <kbd>T</kbd>. |
+| `defaultProfile` | _Required_ | String | Sets the default profile. Opens by typing <kbd>Ctrl</kbd> + <kbd>T</kbd> or by clicking the '+' icon. The guid of the desired default profile is used as the value. |
+| `initialCols` | _Required_ | Integer | The number of columns displayed in the window upon first load. |
+| `initialRows` | _Required_ | Integer | The number of rows displayed in the window upon first load. |
+| `requestedTheme` | _Required_ | String | Sets the theme of the application. Possible values: `"light"`, `"dark"`, `"system"` |
+| `showTerminalTitleInTitlebar` | _Required_ | Boolean | When set to `true`, titlebar displays the title of the selected tab. When set to `false`, titlebar displays "Windows Terminal". |
+| `experimental_showTabsInTitlebar` | Optional | Boolean | When set to `true`, the tabs are moved into the titlebar and the titlebar disappears. When set to `false`, the titlebar sits above the tabs. |
+| **profiles** | | **Array[Object]** | **Properties listed below are specific to each unique profile.** |

```

Comment:
Global above was capitalized. profiles, schemes, and keybindings are not... intentional?

Reply:
Intentional - profiles, schemes, and keybindings are lowercase in the profiles.json file whereas Global isn't included in the file.

Reply:
Ah OK.